src/test-external/index.js

```diff
@@ -38,11 +38,29 @@ const isMonoRepo = (targetDir) => {
   return fs.existsSync(path.join(targetDir, 'lerna.json'))
 }
 
+const hasPackageLock = (targetDir) => {
+  return fs.existsSync(path.join(targetDir, 'package-lock.json'))
```

Comment:
Test for `npm-shrinkwrap.json` too?

Reply:
added in https://github.com/ipfs/aegir/pull/450/commits/02ad679aade8ff5fdbb7c821ac77ba1b17ca18e9